src/commands/build.rs

```diff
@@ -552,6 +552,8 @@ fn sign_images(image_name: &str, tag: Option<&str>) -> Result<()> {
         env::var("SIGSTORE_ID_TOKEN"),
         env::var("GITHUB_EVENT_NAME"),
         env::var("GITHUB_REF_NAME"),
+        env::var("GITHUB_WORKFLOW_REF"),
+        env::var("GITHUB_SERVER_URL"),
```

Comment:
does github provide these?

Reply:
They sure do https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Wait I don't even think I need the SERVER_URL